2.7.0/final.md

```diff
@@ -0,0 +1,958 @@
+
+Please use `LRScheduler.get_last_lr()` to access the learning rate instead.`print_lr` and `verbose` were confusing, not properly documented and were little used, as described in #99270, so we deprecated them in 2.2. Now, we complete the deprecation by removing them completely. To access and print the learning rate of a LRScheduler:
+
+Version 2.6.0
+```
+optim = ...
+lrsched = torch.optim.lr_scheduler.ReduceLROnPlateau(optim, verbose=True)
+// lrsched will internally call print_lr
+```
+
+Version 2.7.0
+```
+optim = ...
+lrsched = torch.optim.lr_scheduler.ReduceLROnPlateau(optim)
+print(lrsched.get_last_lr())
+```
+
+### Change `torch.Tensor.new_tensor()` be on the given Tensor's device by default ([#144958](https://github.com/pytorch/pytorch/pull/144958))
+
+This function was always creating the new Tensor on the "cpu" device and will now use the same device as the current Tensor object. This behavior is now consistent with other `.new_*` methods.
+
+### Please use `torch.export.export_for_training` instead of `capture_pre_autograd_graph` to export the model for pytorch 2 export quantization (#139505)
+
+`capture_pre_autograd_graph` was a temporary API in `torch.export`. Since now we have a better longer term API: `export_for_training` available (starting from PyTorch 2.5), we can deprecate it.
+
+Version 2.6.0
+```python
+from torch._export import capture_pre_autograd_graph
+from torch.ao.quantization.quantize_pt2e import prepare_pt2e
+from torch.ao.quantization.quantizer.xnnpack_quantizer import (
+    XNNPACKQuantizer,
+    get_symmetric_quantization_config,
+)
+quantizer = XNNPACKQuantizer().set_global(
+    get_symmetric_quantization_config()
+)
+m = capture_pre_autograd_graph(m, *example_inputs)
+m = prepare_pt2e(m, quantizer)
+```
+
+Version 2.7.0
+```py
+from torch.export import export_for_training
+from torch.ao.quantization.quantize_pt2e import prepare_pt2e
+# please get xnnpack quantizer from executorch (https://github.com/pytorch/executorch/)
+from executorch.backends.xnnpack.quantizer.xnnpack_quantizer import (
+    XNNPACKQuantizer,
+    get_symmetric_quantization_config,
+)
+quantizer = XNNPACKQuantizer().set_global(
+    get_symmetric_quantization_config()
+)
+m = export_for_training(m, *example_inputs)
+m = prepare_pt2e(m, quantizer)
+```
+
+### Use Manylinux 2.28 and CXX11_ABI=1 for future released Linux wheel builds. 
+With Migration to manylinux_2_28 (AlmaLinux 8 based), we can no longer support OS distros with glibc2_26. These include popular Amazon Linux 2 and CentOS 7. (#143423, ##146200, #148028, #148135, #148195, #148129)
+
+# Deprecations
+
+### Removed CUDA 12.4 support in CI/CD in favor of 12.8 (#148895, #142856, #144118, #145566, #145844, #148602, #143076, #148717)
+### Removed Anaconda support in CI/CD (#144870, #145015, #147792)
+### Dropped support for Python < 3.9 (#147097)
```

Comment:
This is not a deprecation. Also 3.8 was already dead, this is just code cleanup?

Reply:
Should we just delete this then? How do we normally handle platform support shifts?

Reply:
Calling out platform support change can be done in the Infra section. they're very useful.

My point is is that we already dropped support for python < 3.9 a couple releases back. We're not changing platform support in this release.

Reply:
Oh, I see. Removing that item then. Where is this particular Infra section you're referring to?